core/src/test/java/com/linecorp/armeria/common/util/BlockingTaskExecutorBuilderTest.java

```diff
@@ -0,0 +1,22 @@
+package com.linecorp.armeria.common.util;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.Flags;
+
+class BlockingTaskExecutorBuilderTest {
+
+    @Test
+    void testDefault() {
+        final ScheduledThreadPoolExecutor pool =
+                (ScheduledThreadPoolExecutor) BlockingTaskExecutor.builder().build().unwrap();
+        assertThat(pool.allowsCoreThreadTimeOut()).isTrue();
```

Comment:
Could you add more tests that verify overriden configurations?

Reply:
I add it. Thank you!

Reply:
Could you add a test when `keepAliveTime` is '0'?

Reply:
I add simple test. Please check it 😃

Reply:
That looks good.